src/components/pages/search.js

```diff
@@ -79,6 +84,11 @@ function Search({ initialState }) {
       title: t('updateDate'),
       dataIndex: 'updatedOn',
       render: renderUpdatedOn
+    },
+    {
+      title: t('language'),
+      dataIndex: 'language',
+      render: lang => <CountryFlagAndName code={languageData[lang]?.flag} name={languageData[lang]?.name || lang} />
     }
   ];
```

Comment:
It's a small edge case maybe, but is there maybe a quick way to also set a min width for some columns when defining them? 🤔 

Cause of tags it could happen that the other columns get shrunk and for example the language flag + name get wrapped on separate lines 👇 

<img width="1220" alt="Screenshot 2021-11-12 at 10 25 25" src="https://user-images.githubusercontent.com/8189923/141444120-61a18c6b-e93f-4329-8bf5-1df569b04a18.png">

Another quick idea would be to make the space between the icon and text in `CountryFlagAndName` a non breaking space, if that's possible.

Reply:
I've done a fix, not sure how to do it in another way, but it works :).